website/docs/advanced/maintaining-uptime.md

```diff
@@ -0,0 +1,74 @@
+---
+id: maintaining-uptime
+title: Maintain validator uptime with systemd or Docker
+sidebar_label: Maintain Validator Uptime
+---
+
+Validators are expected to maintain connectivity with the Ethereum network 24/7. If you're running your validator on a cloud server, or if you want your validator to automatically start running when the host machine restarts, consider running your client software as a background service. Docker and systemd let you run Prysm as a background service. 
+
+This may be overkill for at-home stakers who use the `prysm.sh` script to run Prysm. To those users, we recommend keeping an eye on your validator. The penalties for occasional downtime are generally negligible. Learn more by reading our [Security Best Practices](../security-best-practices.md).
+
+<Tabs
+  groupId="service"
+  defaultValue="docker"
+  values={[
+    {label: 'Docker', value: 'docker'},
+    {label: 'Systemd', value: 'systemd'},
+  ]
+}>
+<TabItem value="docker">
+
+You can use Docker to run your beacon node and validators as background services.
```

Comment:
This information needs to maintain parity with our docker installation instructions, which could evolve. One recommendation is to instead send people to our install with docker guidelines here rather than inlining commands on this page